src/framework/auth/strategies/default/default-strategy-options.ts

```diff
@@ -18,30 +22,89 @@ export interface NbDefaultStrategyModule {
   defaultMessages?: string[];
 }
 
-// TODO postfix to options
 export interface NbDefaultStrategyReset extends NbDefaultStrategyModule {
   resetPasswordTokenKey?: string;
 }
 
-// TODO postfix to options
-export interface NbDefaultAuthStrategyOptions {
-  baseEndpoint?: string;
-  login?: boolean | NbDefaultStrategyModule;
-  register?: boolean | NbDefaultStrategyModule;
-  requestPass?: boolean | NbDefaultStrategyModule;
-  resetPass?: boolean | NbDefaultStrategyReset;
-  logout?: boolean | NbDefaultStrategyReset;
-  token?: {
-    key?: string;
-    getter?: Function;
+export class NbDefaultAuthStrategyOptions extends NbAuthStrategyOptions {
+  name = 'email';
+  baseEndpoint? = '/api/auth/';
+  login?: boolean | NbDefaultStrategyModule = {
+    alwaysFail: false,
```

Comment:
Why do we need *alwaysFail* in default properties? Maybe we have to move it in some specific Options for tests? I don't think it'll be very useful in production.

Reply:
this is the default strategy, ex-email-pass strategy